packages/ubuntu_welcome/snap/snapcraft.yaml

```diff
@@ -0,0 +1,63 @@
+name: ubuntu-welcome
+version: git
+summary: Ubuntu Welcome
+description: |
+  Ubuntu Welcome description.
+grade: stable
+confinement: devmode
+base: core22
+issues: https://github.com/canonical/ubuntu-desktop-installer/issues
+contact: https://github.com/canonical/ubuntu-desktop-installer/issues
+
+apps:
+  ubuntu-welcome:
+    command: bin/ubuntu_welcome
+    extensions: [gnome]
+    command-chain: [bin/launcher]
+    #desktop: bin/data/flutter_assets/assets/ubuntu-welcome.desktop
+    environment:
+      PATH: $SNAP/usr/bin:$SNAP/bin:$PATH
+      LIBGL_DRIVERS_PATH: $SNAP/usr/lib/$CRAFT_ARCH_TRIPLET/dri
+      GIO_MODULE_DIR: $SNAP/usr/lib/$CRAFT_ARCH_TRIPLET/gio/modules
+      LIVE_RUN: 1
+      LOG_LEVEL: debug
+      SNAP_PYTHON: python3
```

Comment:
i wonder if we could leave out everything else except `LOG_LEVEL` might be nice to keep

- `LIVE_RUN` and `SNAP_PYTHON` are for subiquity
- `PATH`, `LIBGL_DRIVERS_PATH` and `GIO_MODULE_DIR` might not be needed for a non-classic snap?

Reply:
Removed